### ISSUE TITLE
[QUESTION] Where i can edit default inventory items?

### ISSUE THREAD

**Issue by noternob** [GrowtopiaNoobs/GrowtopiaServer#689]
Where i can edit default inventory items?

**Comment by noobesgt** [comment]
search for PlayerInventory

**Comment by playingo** [comment]
yeah @noobesgt, now tell me where to change the stuff :) you obv do not know what to edit u just said "look at PlayerInventory" because you do not know how to actually edit the stuff that is defaultly given.

**Comment by noobesgt** [comment]
Hey, memeingo here? Ok so if ur nerd i can help you, press ctrl-f and place in there PlayerInvetnory , hope i helped you :)

**Comment by playingo** [comment]
@noobesgt skid, do you even know what to change in PlayerInventory thats what i asked. you either resend playerinventory like a skid or change packets

**Comment by noobesgt** [comment]
Please learn the term of skid is

**Comment by playingo** [comment]
script kiddie, which you also are. how does it help to say "ugh just do something in PlayerInventory ugh idk what it is but just do something with it" OMG YES I KNOW THAT IT IS FUCKING PLAYERINVENTORY BUT YOU SHOULD HAVE SAID "yeah u need to change the string packet with a bunch of zeros after the main string which is 16 characters" yeah you should have said something like that not just: "ugh just goto playerinventory and do something" LOL HOW IS THAT HELPFUL WHAT YOU SAID YOU KID?

**Comment by noobesgt** [comment]
Calm down, get a new life of stealer

**Comment by Secret-8179** [comment]
Oh shit man i dont even know whos playingo i heard about you bro but i never talked with you before i saw you just in the github nice to met you bro! @playingo
